src/main/java/seedu/address/MainApp.java

```diff
@@ -135,7 +135,24 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             initialOrderBook = new OrderBook();
         }
 
-        return new ModelManager(initialAddressBook, initialTaskBook, initialOrderBook, userPrefs);
+        ModelManager modelManager = new ModelManager(initialAddressBook, initialTaskBook, initialOrderBook, userPrefs);
+        try {
+            modelManager.checkClientAndOrderRelation();
+        } catch (DataConversionException e) {
+            logger.info("Data file not corrupted. Will be starting with a empty OrderBook");
```

Comment:
Since data file corruption is a big problem, is it better to use `logger.warning` here? 
The error message is not very clear, could you try improving it?